pkg/cmd/roachtest/roachtestutil/mixedversion/planner.go

```diff
@@ -418,11 +427,22 @@ func (p *testPlanner) Plan() *TestPlan {
 		isLocal:        p.isLocal,
 	}
 
-	// Probabilistically enable some of of the mutators on the base test
-	// plan generated above.
+	failureInjections := make(map[string]struct{})
+	for _, m := range failureInjectionMutators {
+		failureInjections[m.Name()] = struct{}{}
+	}
+
+	// Probabilistically enable some of the mutators on the base test
+	// plan generated above. We disable any failure injections that
```

Comment:
This is fine for now, but I wonder if it'd be cleaner to move this to some sort of `IsValidMutator` method in the `mutator` interface once you start adding more mutators with different criteria.